modules/server/src/graphqlRoutes.js

```diff
@@ -243,7 +243,7 @@ const createEndpoint = async ({ esClient, graphqlOptions = {}, mockSchema, schem
 	return router;
 };
 
-const createSchemasFromConfigs = async ({
+export const createSchemasFromConfigs = async ({
```

Comment:
🙏 I knew we'd need this one at some point, just couldn't remember which was the one